Frontend/app/tests/example.ts

```diff
@@ -0,0 +1,31 @@
+// A sample QUnit test
+
+/*QUnit.test("equal test", function (assert) {
+  assert.equal( 0, 0, "Zero, Zero; equal succeeds" );
+  assert.equal( "", 0, "Empty, Zero; equal succeeds" );
+  assert.equal( "", "", "Empty, Empty; equal succeeds" );
+  assert.equal( 0, false, "Zero, false; equal succeeds" );
+});*/
+
+var participants = require("../pages/enterprises/participants.js");						//Access Participant variables and functions
```

Comment:
My mistake. I did testing with the existing code we wrote for the Prototype to see how it can be used to access variables and functions from other files. I should have left it out since we haven't done any of this for the main app yet.